internal/schema/type_test.go

```diff
@@ -19,30 +19,35 @@ func TestType_GetQueryFieldsString(t *testing.T) {
 	cases := map[string]struct {
 		TypeName string
 		Depth    int
+		Mutation bool
```

Comment:
Is there a `true` case we want to test here also?

Reply:
probably....